packages/components/form-control/tests/form-control.test.tsx

```diff
@@ -59,73 +59,56 @@ describe("<FormControl />", () => {
         </FormControl>
       </>,
     )
-    expect(
-      screen
-        .getByTestId("FormControl-a")
-        .getElementsByClassName("ui-form__error-message")[0],
-    ).toHaveTextContent("Email is required.")
-    expect(
-      screen
-        .getByTestId("FormControl-b")
-        .getElementsByClassName("ui-form__helper-message")[0],
-    ).toHaveTextContent("Please enter your email")
+    expect(screen.getByText("Email is required.")).toBeInTheDocument()
+    expect(screen.queryByText("Please enter your email")).toBeInTheDocument()
```

Comment:
[NITS] For testing the existence of elements, prefer using `getByText` to `queryByText`.